templates/Dockerfile.template

```diff
@@ -20,6 +20,7 @@ RUN set -ex; \
 
 {{- $gosu := index .Packages "gosu" }}
 {{- $cassandra := index .Packages "cassandra" }}
+{{- $c2dRelease := index .TemplateArgs "c2dRelease" }}
```

Comment:
Why we decide to add additional template parameter instead use ENV C2D_RELEASE=${CASSANDRA_VERSION} in dockerfile?

Reply:
Because cassandra packages do not always follow semantic versioning which is required by deployer, that is why we need to explicitly add this variable